### ISSUE TITLE
Git workflow

### ISSUE THREAD

**Issue by bsmithyman** [uwoseis/zephyr#37]
Hi @ShaunHadden, @gerhardpratt.

I've created a branch called `dev`, and it now has the multiscale code merged in. It seems to work okay.

From now on, I would like to restrict how we merge to `master`. We should do all of our integration work on the `dev` branch, and then do a well-tested & discussed pull request from `dev` to `master` when we're 100% convinced that everything works.
- For bug fixes that affect `master`, we should do a pull request from a branch called `fix/nameOfBug` to `master`, and then merge from `master` to `dev`.
- For bug fixes that don't affect `master`, we should do a pull request or direct fix on `dev`.
- For new branches, we should branch off of `dev`, or `master`, but always do the merge PR on `dev`

**Comment by bsmithyman** [comment]
@ShaunHadden, can you commit your various stuff to `ani_testing`, and then set up a PR from  `ani_testing` to `dev`? Possibly there's nothing new to push, but I would like to clear up that branch. In particular, I would like to get your various testing notebooks and any outstanding code fixes checked in either:
1. On the Zephyr repository, or
2. On a private repository under your name

That way there's some protection against errors & changes that we don't notice. If they're not nice and tidy, I would suggest moving them to a different folder (`codebases/rootdir/util`?), and making that a private repository. Exactly what goes where can be something that you decide (w/ input from @gerhardpratt and me, if you want it).

I would like notebooks in Zephyr to be:
1. Changing infrequently (for file size reasons)
2. Relatively well-documented and stable

I'm doing some cleanup on my end too :)

**Comment by ShaunHadden** [comment]
Hey @bsmithyman ,

When attempting to set up this PR it says:

There isn't anything to compare. dev is already up-to-date with all commits from ani_testing.

Does this mean ani_testing is in the clear to be closed?

Let me know,
Shaun

**Comment by bsmithyman** [comment]
Yep, at least as far as what's on the server. The question is whether there's anything left to push from your computer (charon). If not, we may as well delete the branch and start working off of `dev`.

**Comment by ShaunHadden** [comment]
Upon performing a 'git diff', the only differences to source code are white space and/or comments. to be certain, I reverted any changes on my local machine and re-ran the 'test fullwv' script and all 3 still match up (I think we concluded that the only concrete change required was the .conjugate() addition to the source term in the extend.py extension?) Therefore, I believe you can close the branch!

**Comment by bsmithyman** [comment]
Perfect, thanks Shaun.
